unlock-app/src/components/interface/checkout/ReturningButton.tsx

```diff
@@ -24,7 +24,7 @@ export const ReturningButton = forwardRef(
       onClick,
       ...restProps
     } = props
-    const [state] = useActor(checkoutService)
+    const state = useSelector(checkoutService, (s) => s)
```

Comment:
This is a bit strange. Since we only use the `context.paywall` value, maybe the selector function can just return that?